ccvm_simulators/solvers/dl_solver.py

```diff
@@ -298,14 +298,12 @@ def _solve(
                 * np.sqrt(dt)
                 / noise_ratio_i
             )
-            c += (
-                dt * c_drift
-                + 2 * g * torch.sqrt(c**2 + s**2 + 0.5) * wiener_increment_c
-            )
-            s += (
-                dt * s_drift
-                + 2 * g * torch.sqrt(c**2 + s**2 + 0.5) * wiener_increment_s
-            )
+            
+            diff = 2 * g * torch.sqrt(c**2 + s**2 + 0.5) 
```

Comment:
Is diff short for difference? If so, difference between what?

Reply:
I believe diff is short for diffusion? You can maybe call it `diffusion` or `diffusion_factor`.